src/pybind/mgr/dashboard_v2/controllers/rbd.py

```diff
@@ -68,3 +84,29 @@ def get(self, pool_name):
         if status == ViewCache.VALUE_EXCEPTION:
             raise value
         return {'status': status, 'value': value}
+
+    def create(self, data):
+        if not self.rbd:
+            self.rbd = rbd.RBD()
+
+        # Get input values
+        size = data.get('size', 4 * 1024 ** 3)
+        obj_size = data.get('obj_size', 2 ** 22)
```

Comment:
please don't set a default value to `obj_size` and let librbd python binding take care of that. If `obj_size` is not specified then `order = None`.

Reply:
Okay